swagger/swagger_test.go

```diff
@@ -259,3 +266,229 @@ func TestNew(t *testing.T) {
 		require.Equal(t, "success", string(bodyBytes))
 	})
 }
+
+func TestNewWithFileContent(t *testing.T) {
```

Comment:
This is passing but not using FileContent, we need an different swagger.json file to verify the data we get back from `/swagger` matches the content.

Reply:
>This is passing but not using FileContent

I don't follow. Almost every subtest sets FileContent. For example:

```go
func TestNewWithFileContent(t *testing.T) {
	t.Run("Endpoint check with only custom path", func(t *testing.T) {
		app := fiber.New()

		cfg := Config{
			Path:        "custompath",
			FileContent: swaggerJSON,
		}
```

>we need an different swagger.json file to verify the data we get back from /swagger matches the content.

Why does TestNewWithFileContent need to do that, but not TestNew? Also, why would we need a different file for that?

Reply:
@willfaught Lets try something, in the `NewWithFileContent` tests, set FilePath to `myswagger.json`. Internally this should expose the spec on `/myswagger.json`, but use the embed swagger.json as the content.

If that passes we should be good to go

Reply:
[ac66002](https://github.com/gofiber/contrib/pull/1085/commits/ac66002a48f4eddedd2e70129e1dbba524d04196)

Reply:
Awesome, it passed